### ISSUE TITLE
Clarification on Banner Ad Refresh Behavior after OnBannerAdLoadFailed

### ISSUE THREAD

**Issue by alireza-sunday** [googleads/googleads-mobile-unity#3626]
### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2022.3.27f1
  * Google Mobile Ads Unity plugin version: v9.0.0
  * Plugin installation method: unitypackage import
  * Platform: iOS, Android

### [REQUIRED] Step 2: Describe the problem

**Description**:

We are encountering a behavior related to banner ads and would like clarification regarding the `OnBannerAdLoadFailed` event. According to the documentation, this callback is triggered when a banner ad fails to load into the banner view. However, there is also a built-in refreshing mechanism for banner ads that reloads ads based on a predefined refresh rate.

**Question**:

When the `OnBannerAdLoadFailed` event is invoked, does the automatic banner ad refresh mechanism still function as expected, or do we need to manually handle the banner ad reloading process? Specifically:

If a banner ad fails to load and `OnBannerAdLoadFailed` is called, should we create a new instance of the banner ad and load it manually, or will the built-in refresh mechanism continue to work and handle subsequent ad loading attempts automatically?

**Comment by NVentimiglia** [comment]
@alireza-sunday 

Yes, the banner refresh mechanism will function even after `OnBannerAdLoadFailed`.

**Comment by Menyus777** [comment]
@NVentimiglia 
And will this event trigger when in case of refresh there is an error and an Ad can't be loaded?
E.g.:

1. Banner loads successfully
2. At first refresh banner fails to refresh
3. Is OnBannerAdLoadFailed triggers?

**Comment by NVentimiglia** [comment]
Correct. This error is raised for all failure to load errors.
